tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -213,6 +227,11 @@ def generate_files(list, args):
                           '" target="lib\\netstandard2.0\\Microsoft.AI.MachineLearning.Interop.pdb" />')
 
     # Process runtimes
+    # Process linux
+    if (args.linux_build != 'None'):
+        files_list.append('<file src=' + '"' + os.path.join(args.sources_path, args.linux_build) +
```

Comment:
>linux_build [](start = 84, length = 11)

I think we can just hard-code the shared library name - libonnxruntime.so. I think we have used onnxruntime.dll in a hard-coded fashion in a lot of places.

Reply:
Unless there should be some functionality to copy the file from wherever it is to the correct location. 

---
In reply to: [465305444](https://github.com/microsoft/onnxruntime/pull/4703#discussion_r465305444) [](ancestors = 465305444)